### PR TITLE
New Styles using TailwindCSS

### PR DESCRIPTION
**Pull request by mkasberg**
@JohnRDOrazio I've prototyped a few things; I'd love to get your initial thoughts and feedback. This is _not_ complete (in fact most pages look broken), I've just done some simple styles on index.php and the navbar, I'd like to get some early feedback before going too much further.

* See my changes to the README to see how local dev works.
* I think we don't have enough pages to warrant top nav and side nav, I've prototyped only top nav.
* Colors are very easy to change, the screenshot is the "Autumn" color palette.
* This isn't a theme or template, this is just TailwindCSS with some light styling from scratch, so it's easy to customize anything to our liking. I do think that the relative simplicity of the style in this PR is less overwhelming to new users, I'm curious what you think...

I'm looking for feedback along the lines of either:
* This is good, keep going, and we can continue to make tweaks and change colors.
* OR, The TailwindCSS approach is good but we need to style things significantly differently.
* OR, The sb-admin theme was better, let's stop pursuing a TailwindCSS approach and instead find a way to customize sb-admin to our liking without such significant UI changes.

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,18 @@ once the Diocesan Calendar is defined, it can be requested using the `diocesanCa
 
 The National and Diocesan Calendar data can be defined directly through the interfaces offered by this frontend.
 
-# Localization of the Frontend
+## Development
+
+The HTML is plain PHP. (It could be served locally with `php -S localhost:8000`.) (You'll need php-intl installed.)
```
Comment (JohnRDOrazio) [inline README.md:23]: I would rephrase this something along the lines of "The pages consist of plain HTML and PHP, without any build steps" if that's what you're trying to convey

File: layout/header.php [hunk 2]
```diff
@@ -15,126 +15,72 @@
...
-<nav class="sb-topnav navbar navbar-expand navbar-light bg-white shadow">
-    <!-- Navbar Brand -->
-    <a class="navbar-brand ps-3" href="/">Navbar</a>
 
-    <!-- Sidebar Toggle (Topbar) -->
-    <button class="btn btn-link btn-sm order-1 order-lg-0 me-4 me-lg-0" id="sidebarToggle"><i class="fas fa-bars"></i></button>
+<!-- Navbar -->
+<nav class="navbar fixed top-0 start-0 z-10 w-full bg-base-200 border-b border-gray-400">
+    <div class="navbar-start">
+        <a class="text-xl font-bold text-primary" href="/">Litcal.org</a>
```
Comment (JohnRDOrazio) [inline layout/header.php:22]: Here I would put just `LitCal`, not the domain, seeing the situation with the domain isn't quite clear. I'm still just using https://litcal.johnromanodorazio.com and https://litcal-staging.johnromanodorazio.com for the main and dev branches, until there's a clearer situation with the litcal.org domain... I have also created a logo we could put alongside here, I'll check it into the repo

File: README.md (hunk 1 above)
Comment (JohnRDOrazio) [reply README.md:23]: Perhaps we should also mention the minimum required PHP version? Currently it is PHP 7.4

**Comment by JohnRDOrazio** [comment]
I do like this approach, it means we don't have to worry about things changing drastically when bootstrap decides to change it's classes or sb-admin decides to change the default colors. We're not locked in, we can have greater consistency and less worries about things breaking. At least I hope :)
Perhaps we can still use cards, so that the interface doesn't change too much? At least as a first transition. I like `daisy-ui` that makes for much shorter css classes, not dissimilar from bootstrap, and offers many similar components to bootstrap.
Do you think we should split the main API into two different sections, "Calendar to retrieve" and "Customizable API"? I figure it's more straightforward to have them in the same "card" as before, but I'm open to hearing other opinions.

**Comment by JohnRDOrazio** [comment]
Of course this is still a rough draft, moving forwards we will of course need to add some padding around the borders of the page 😄

**Comment by mkasberg** [comment]
> Perhaps we can still use cards, so that the interface doesn't change too much? At least as a first transition.

IMO, if we want to maintain that much similarity with the current design, it makes more sense to me to continue using sb-admin as our foundation. I've opened another PR that prototypes doing just that: #124 Take a look whenever you have time and let me know your thoughts here or on Slack.

> Do you think we should split the main API into two different sections, "Calendar to retrieve" and "Customizable API"?

I do. I don't feel too strongly and I think it's fine either way, but I found it confusing to have the dropdowns below the button override (remove) the dropdowns above the button.
